app/boards/shields/romac_plus/romac_plus.conf

```diff
@@ -0,0 +1,3 @@
+# Uncomment to enable encoder
+CONFIG_EC11=y

```

Comment:
This is likely the source of the current build failures. I don't think I've ever tested the encoder stuff on stm32. Is the encoder *always* on this RoMac v4? If not, can you comment these out?

Reply:
Got it. For the RoMac+ v4 users have the option of an encoder or a key. So what I will do is comment out those configs as requested.

Reply:
Alright, I disabled both the EC11 configs for consistency's sake since that's what I saw in other shields. Still builds. I'll just watch the build tests.